app/assets/stylesheets/blacklight.scss

```diff
@@ -66,3 +66,37 @@
     }
   }
 }
+
+dt.blacklight-full_text_tesimv {
+  @extend .h4;
+  display: inline-block;
+  float: none;
+  text-align: initial;
+  width: auto;
+
+  &[data-toggle="collapse"]:after {
+    /* symbol for "opening" panels */
+    font-family: 'Glyphicons Halflings';  /* essential for enabling glyphicon */
```

Comment:
Use `//` comments everywhere<br>Properties should be ordered color, content, float, font-family, font-size, line-height, padding-left<br>Prefer double-quoted strings